devtools_options.yaml

```
@@ -1,3 +1,3 @@
-description: This file stores settings for Dart & Flutter DevTools.
-documentation: https://docs.flutter.dev/tools/devtools/extensions#configure-extension-enablement-states
-extensions:
+description: This file stores settings for Dart & Flutter DevTools.
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)